iceoryx_examples/iceperf/base.cpp

```diff
@@ -16,54 +17,55 @@
 #include "base.hpp"
 
 
-void IcePerfBase::prePingPongLeader(uint32_t payloadSizeInBytes) noexcept
+void IcePerfBase::preLatencyPerfTestLeader(const uint32_t payloadSizeInBytes) noexcept
 {
-    sendPerfTopic(payloadSizeInBytes, true);
+    sendPerfTopic(payloadSizeInBytes, RunFlag::RUN);
 }
 
-void IcePerfBase::postPingPongLeader() noexcept
+void IcePerfBase::postLatencyPerfTestLeader() noexcept
 {
     // Wait for the last response
     receivePerfTopic();
 }
 
 void IcePerfBase::releaseFollower() noexcept
 {
-    sendPerfTopic(sizeof(PerfTopic), false);
+    sendPerfTopic(sizeof(PerfTopic), RunFlag::STOP);
 }
 
-double IcePerfBase::pingPongLeader(uint64_t numRoundTrips) noexcept
+double IcePerfBase::latencyPerfTestLeader(const uint64_t numRoundTrips) noexcept
```

Comment:
Sorry that I am seeing this just now but this would be the perfect place to return `units::Duration`. Otherwise no one knows that you are returning here some latency in microseconds.